### DISCUSSION TITLE
20240422-be-part/

### DISCUSSION THREAD

**Discussion by giscus** [inflearn/tech-blog-comment#55] [Comments]
# 20240422-be-part/

인프랩 채용 공고 안녕하세요. 인프랩의 향로입니다. 저희 인프랩 개발팀은 현재 백엔드, 프론트엔드, 모바일, 데브옵스 파트로 나뉘어져 있습니다. 각각의 개발 파트가 어떻게 일 하…

https://tech.inflab.com/20240422-be-part/

**Comment by sh5080** [comment]
신규 시스템 스택 - Nest.js 의 Multi Module 을 클릭하면 우아한 형제들의 기술블로그가 개편되어서 접속하자마자 바로 다른 페이지로 넘어가는데, 혹시 https://techblog.woowahan.com/2637/ 의 내용이 동일한 내용일까요?

**Comment by dawinkweon** [comment]
CTO님 유튜브 영상들보고 방문해요~ Enterprise에서만 일해온 외국 개발자인데,Scale-up과정에서도 Mature해보이는 Tech stack과 Development Strategy이 여러가지로 마음에 드네요, 이상적인 회사와 리더십 같아보입니다. 물론 인프런 플랫폼도 너무 좋아요. ATB for recruitment!

**Reply by dawinkweon** [reply]
의도적으로 생략된 것 같지만, IaC와 가볍게 언급된 Datadog와 관련 Observability랑 on-call paging은 인프랩에서 어떻게 하는지 궁금하네요.
